Core.ApplicationServices/SystemUsage/IItSystemUsageService.cs

```diff
@@ -63,11 +63,16 @@ public interface IItSystemUsageService
         /// <summary>
         /// Edits a system relation
         /// </summary>
-        /// <param name="fromSystemUsageId">Id of the "source" it system usage</param>
-        /// <param name="relationId">Id of the specific relation in the "source" system usage</param>
-        /// <param name="toSystemUsageId">Id of the "target" it system usage</param>
-        /// <param name="targetInterfaceId">Id of the specific "target" system usage interface to be used by the "source" system usage</param>
+        /// <param name="fromSystemUsageId">Id of the "from" it system usage</param>
+        /// <param name="relationId">Id of the specific relation in the "from" system usage</param>
+        /// <param name="toSystemUsageId">Id of the "to" it system usage</param>
+        /// <param name="description">Updated description</param>
+        /// <param name="reference">Updated reference</param>
+        /// <param name="toInterfaceId">Id of the specific "to" system usage interface to be used from the "from" system usage list of available interfaces</param>
+        /// <param name="toContractId">Id of the specific "to" system usage contract to replace the existing system relation value</param>
+        /// <param name="toFrequencyTypeId">Id of the specific "to" system frequency type to replace the existing system relation value</param>
         /// <returns></returns>
-        Result<SystemRelation, OperationError> ModifyRelation(int fromSystemUsageId, int relationId, int toSystemUsageId, int? targetInterfaceId = null);
+        Result<SystemRelation, OperationError> ModifyRelation(int fromSystemUsageId, int relationId, int toSystemUsageId, string description, string reference,
+            int? toInterfaceId = null, int? toContractId = null, int? toFrequencyTypeId = null);
```

Comment:
Når de er markeret som optional, så indikerer det at der ikke sker noget hvis de er null. Det fungerer bare ikke når null er en gyldig værid for både toInterfaceId, toContractId og toRFrequencyType

De skal ikke være optional men required nullable.